packages/webapp/src/inductions/components/induction-step-video.tsx

```diff
@@ -34,20 +39,31 @@ export const InductionStepVideo = ({
 }: Props) => {
     const [ualAccount] = useUALAccount();
     const [submittedVideo, setSubmittedVideo] = useState(false);
+    const [videoSubmissionPhase, setVideoSubmissionPhase] = useState<
+        VideoSubmissionPhase | undefined
+    >(undefined);
 
-    const submitInductionVideo = async (videoHash: string) => {
+    const submitInductionVideo = async (videoFile: File) => {
         try {
+            setVideoSubmissionPhase("uploading");
+            const videoHash = await uploadToIpfs(videoFile);
+
             const authorizerAccount = ualAccount.accountName;
             const transaction = setInductionVideoTransaction(
                 authorizerAccount,
                 induction.id,
                 videoHash
             );
             console.info(transaction);
+            setVideoSubmissionPhase("signing");
             const signedTrx = await ualAccount.signTransaction(transaction, {
-                broadcast: true,
+                broadcast: false,
             });
             console.info("inductvideo trx", signedTrx);
+
+            setVideoSubmissionPhase("finishing");
+            await uploadIpfsFileWithTransaction(signedTrx, videoHash);
+
             setSubmittedVideo(true);
         } catch (error) {
             onError(error, "Unable to set the induction video");
```

Comment:
We need to `setVideoSubmissionPhase()` back to undefined here. Otherwise, if an error occurs or the user chooses not to sign, we don't get stuck like this:

![image](https://user-images.githubusercontent.com/7911424/118680944-2d3bbb00-b7cd-11eb-9f27-e5e2960d3e08.png)

Reply:
I pushed the fix.